client/src/pages/Editor/Editor.Layout.jsx

```diff
@@ -22,19 +52,121 @@ export default function EditorTab() {
     setEditorCode(() => (value ? value : ""));
   }
 
+  const onFontSizeChange = (operator) => {
+    const increment = operator === "+" ? 10 : -10;
+
+    if (
+      (operator === "+" && fontSize < TEXT_EDITOR_MAX_FONT_SIZE) ||
+      (operator === "-" && fontSize > TEXT_EDITOR_FONT_SIZE)
+    ) {
+      const newFontSize = fontSize + increment;
+      const newLineHeight = lineHeight + increment;
+      setFontSize(newFontSize);
+      setlineHeight(newLineHeight);
+      setItem(FONT_SIZE, newFontSize);
+      setItem(LINE_HEIGHT, newLineHeight);
+    }
+  };
+
+  const onDownloadCode = () => {
+    const link = document.createElement("a");
+    const getInput = editorCode;
+    const file = new Blob([getInput], { type: "text/html" });
+    link.href = URL.createObjectURL(file);
+    link.download = "file";
+    link.click();
+    URL.revokeObjectURL(link.href);
+  };
+
+  async function copyToClipboard(text) {
+    await navigator.clipboard.writeText(text);
+    toast("Code Copied to Clipboard");
+  }
+
   return (
     <>
       <NavBar />
-      <Editor
-        height="94dvh"
-        language={selectedLanguage}
-        value={codeSave ? codeSave.code : ""}
-        onChange={preTextRemove}
-        theme="vs-dark"
-        options={{
-          //readOnly: !hasUser,
-          lineHeight: 20,
-        }}
+      <MainDiv>
+        <Editor
+          height="calc(100vh - 70px)"
+          // height="94dvh"
```

Comment:
@Navdeep-av - Remove commented if not in use

Reply:
@xDAnkit Updated